src/gui/HelpWidget.cpp

```diff
@@ -54,3 +58,14 @@ HelpWidget::HelpWidget() : QVBoxWidget() {
 
 
 }
+
+void HelpWidget::onLinkClicked(const QString& link) {
+    if (link == "Ctrl+N") {
+        emit ctrlNPressed();
+    }
+    if (link == "Ctrl+O") {
```

Comment:
Make it `else if`, no unnecessary checks should be going once a condition becomes true.